src/utils/waitFor.ts

```diff
@@ -0,0 +1,42 @@
+interface waitForEventDescription {
+	eventCheck: () => boolean;
+	maxWaitTime?: number;
+	intervalTime?: number;
```

Comment:
Is it possible to consider renaming these properties to include the time unit instead of using the 'Time' suffix? For instance, `maxWaitMilliseconds`?

Reply:
Let's see if after recent JWP changes we'll still need this interface and function 😈